### DISCUSSION TITLE
i cant properly extract the zip

### DISCUSSION THREAD

**Discussion by PillowzHere** [Novum/vkQuake#617] [Q&A] [answered]
when i try to extract the zip using the zip extractor windows comes with, it says the folder is empty (see file below):
![what](https://user-images.githubusercontent.com/123040830/213338331-e09316f5-4cfc-4278-b5b5-a478a5b12b65.png)

when i try to open the folder to see if it's actually empty, the following message appears:
![let me in dammit](https://user-images.githubusercontent.com/123040830/213338349-ecf708f6-688d-4acf-a519-5ddb73d252e8.png)

when i try to extract it using 7-Zip, it seems fine but about half of the files are missing and i get this error message:
![jesus christ not even 7-zip works](https://user-images.githubusercontent.com/123040830/213338528-8aa3c595-800e-400f-b656-672f87b844db.png)

please help guys

**Comment by Novum** [comment]
And where exactly did you get this zip file from?

**Comment by temx** [comment, marked as answer]
That filename looks like what you get by using GitHub's "Download ZIP" option, which gives you the source code. The error message implies that the download was truncated at ~73 KB.
You probably want to download a release from https://github.com/Novum/vkQuake/releases and check the file size matches what it says on the page before extracting.

**Reply by PillowzHere** [reply]
ah ok, thank you

**Reply by garoto** [reply]
> The error message implies that the download was truncated at ~73 KB.

How do you figure that number from the screenshots?

**Reply by micwoj92** [reply]
compressed size in screenshot
